test/requests/on_type_formatting_test.rb

```diff
@@ -154,6 +154,39 @@ def test_pipe_is_removed_if_user_adds_manually_after_completion
     assert_equal(expected_edits.to_json, T.must(edits).to_json)
   end
 
+  def test_pipe_is_removed_if_user_adds_manually_after_block_argument
+    document = RubyLsp::Document.new(source: +"", version: 1, uri: "file:///fake.rb")
```

Comment:
We made a recent change to standardize URI handling, so this needs to become
```suggestion
    document = RubyLsp::Document.new(source: +"", version: 1, uri: URI("file:///fake.rb"))
```